src/Contracts/TenantDatabaseManager.php

```diff
@@ -6,6 +6,13 @@
 
 interface TenantDatabaseManager
 {
+    /**
+     * Return the config key that separates databases (e.g. 'database' or 'schema').
+     *
+     * @return string
+     */
+    public function getSeparator(): string;
```

Comment:
We don't need a weird `separate_by` config now. We just use this to know what key to change in the connection config array.